File: example/ApiTest/JavaScript/converse/1_converse_chat_assistant.js
const myHeaders = new Headers();
myHeaders.append("Content-Type", "application/json");
myHeaders.append("Authorization", "Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm");

const raw = JSON.stringify({
  "question": "你好，你是谁?",
  "stream": true
});

const requestOptions = {
  method: "POST",
  headers: myHeaders,
  body: raw,
  redirect: "follow"
};

fetch("http://127.0.0.1/api/v1/chats/7896ac36aef011efa8e70242ac120003/completions", requestOptions)
  .then((response) => response.text())
  .then((result) => console.log(result))
  .catch((error) => console.error(error));

File: example/ApiTest/Python/converse/1_converse_chat_assistant.py
import requests
import json

url = "http://127.0.0.1/api/v1/chats/7896ac36aef011efa8e70242ac120003/completions"

payload = json.dumps({
  "question": "你好，你是谁?",
  "stream": True
})
headers = {
  'Content-Type': 'application/json',
  'Authorization': 'Bearer ragflow-hjNzA4ODI4YWM5MTExZWY5YzUyMDI0Mm'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)
